homeassistant/components/tile/device_tracker.py

```diff
@@ -111,12 +114,12 @@ def longitude(self) -> float:
     @property
     def name(self):
         """Return the name."""
-        return self._tile.name
+        return f"{self._entry.data[CONF_USERNAME]}: {self._tile.name}"
 
     @property
     def unique_id(self):
         """Return the unique ID of the entity."""
-        return f"tile_{self._tile.uuid}"
+        return f"tile_{self._entry.data[CONF_USERNAME]}_{self._tile.uuid}"
```

Comment:
I think you should add unique ID migration. Look [here](https://github.com/home-assistant/core/blob/dev/homeassistant/components/nam/sensor.py#L46-L57).

Reply:
Ooh, great idea! Thanks!